charmcraft/application/commands/__init__.py

```diff
@@ -117,6 +118,7 @@ def fill_command_groups(app: craft_application.Application) -> None:
         [
             Analyse,
             Analyze,
+            RemoteBuild,
```

Comment:
Should this count as Lifecycle?

Reply:
It looks like that's where Snapcraft has it, so I've adjusted.